src/store/reducers/devicesReducer.js

```diff
@@ -58,21 +73,22 @@ const fetchableTypes = [
 
 export const redux = createReducerScenario({
   updateDevices: { type: 'DEVICES_UPDATE', reducer: updateDevicesReducer },
-  registerError: { type: 'DEVICES_REGISTER_ERROR', reducer: errorReducer },
+  registerError: { type: 'DEVICES_REDUCER_ERROR', reducer: errorReducer },
   isFetching: { multiType: fetchableTypes, reducer: pendingReducer },
 });
 
 export const reducer = { devices: redux.getReducer(initialState) };
 // ========================= Reducers - END
 
 // ========================= Selectors - START
-export const accessReducer = state => state.devices;
-export const getEntities = state => accessReducer(state).entities;
-export const getItems = state => accessReducer(state).items;
+export const getDevicesReducer = state => state.devices;
+export const getEntities = state => getDevicesReducer(state).entities;
```

Comment:
getEntities and getItems doesn't have Devices in the name. Should they?

When would we use those instead of getDevices?

Reply:
I don't think they should. They are really just there to help inside the devicesReducer (notice how they are being used later in this file).

Reply:
If getEntities and getItems aren't used except internally, we should remove the export from them.